### PR TITLE
Creating a Guides section and consolidating content

### PR DESCRIPTION
**Pull request by ned1313**
The Guides section provides examples of how to accomplish different tasks in Gloo, and the guides will link back to the Concepts pages that explain the what and why. Operations will be sparse until docs can be written about the maintenance and operations of Gloo.

Developer Guides will be moved under Guides for consistency. Security, Observability, Knative with Gloo, and Gloo Integrations will all be moved under Guides with pages or sections in Concepts as necessary.

### CODE REVIEW

File: docs/content/getting_started/_index.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ helm install gloo gloo/gloo --namespace gloo-system
 {{< /tab >}}
 {{< /tabs >}}
 
-That's it! With Gloo installed you can try our [Hello World example]({{% versioned_link_path fromRoot="/gloo_routing/hello_world/" %}}) to get an idea of how Gloo can be used.
+That's it! With Gloo installed you can try our [Hello World example]({{% versioned_link_path fromRoot="/guides/traffic_management/hello_world/" %}}) to get an idea of how Gloo can be used.
```
[the remark this answers is not in the document]
Comment (ned1313) [reply docs/content/getting_started/_index.md]: Strange, it's working for me.

[the remark this answers is not in the document]
Comment (kdorosh) [reply docs/content/getting_started/_index.md]: works for me after i pulled most recent changes

**Comment by ned1313** [comment]
Links are all fixed and placeholders have been updated.

**Pull request merged**